src/dmd/func.d

```diff
@@ -351,7 +352,7 @@ extern (C++) class FuncDeclaration : Declaration
      */
     ObjcFuncDeclaration objc;
 
-    extern (D) this(const ref Loc loc, const ref Loc endloc, Identifier ident, StorageClass storage_class, Type type)
+    extern (D) this(const ref Loc loc, const ref Loc endloc, Identifier ident, StorageClass storage_class, Type type, bool noreturn = false)
```

Comment:
Could this instead be `FUNCFLAGS flags`?  I imagine that there's going to be a few more attribute-like information that can be transposed to D - always_inline, noinline, nothrow, pure? - that will require the same propagation in syntax copies.

Reply:
I did it this way to avoid importing dmd.func, the same as successfully avoiding importing dmd.target.

Reply:
Storage classes are already provided for many of them.